src/ufoLib2/objects/font.py

```diff
@@ -515,4 +553,6 @@ def save(
             if tmp is not None:
                 tmp.close()
 
-        self._path = path
+        # Only remember path if it isn't a fs.base.FS.
```

Comment:
Why can't you remember a `fs.base.FS`? If I understand the documentation correctly it's just a filesystem folder?

Reply:
Private talks with @anthrotype -- it's complicated :D read: i don't understand.

Reply:
Oh, yes: it would mean we assign the fs class to Font._path, and we weren't sure we wanted that.

Reply:
what about asking this `FS` think for a path-like something that you can remember here?

Reply:
Hm. maybe it would be easiest if we accepted fs.base.FS as a valid PathLike.

Reply:
because not all FS objects are OsFS with a corresponding filesystem path. E.g. think about MemoryFS.

If you want you can call getsyspath("") method of OsFS object and set that to self._path. But you then have to catch the `fs.errors.NoSysPath` and skip if the FS object does not implement a filesystem path.